README.md

```diff
@@ -1,2 +1,188 @@
+<a name="readme-top"></a>
+
+<div align="center">
+    <br/>
+
 # TDD-Project-Ruby
- "Embark on a test-driven development journey with this project. Create a class using TDD principles, writing tests before code. Your commit history will showcase the proper sequence, ensuring a robust and well-documented solution."
+
+![e2954fb3deb2407e8e4bcb40b1fd6376](https://github.com/AnsarIbrahim/LeaderBoard-new/assets/117971223/9d715e43-a095-4b33-b5f2-161236e8fdcc)
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Kindly remove the link to the `Live Demo` section as it points to a section that has been removed making this link broken